src/website/views/components/forms/range-slider.pug

```diff
@@ -37,3 +42,239 @@ h4 With Labels
       <input class="a-input -mx--2" type="range">
       <label class="-text">5TB</label>
     </div>
+
+h3 Achieve consistent rendering
+
+p.-text
+  | As webkit browsers doesn't support a two different sections for the track of the slider, we need to create our own
+  | elements to change left side color of the track. You will have to develop a JavaScript code for moving
+  | <code>a-input__thumb</code> element and changing size to <code>a-input__progress</code> element, or
+  | use Chi JavaScript library as it provides this functionality out of the box.
+
+h4 Base
+.example.-mb--3
+  .-p--3
+    .a-inputWrapper
+      input(class="a-input", type="range")
+      .a-input__progress
+      .a-input__thumb
+  :code(lang="html")
+    <div class="a-inputWrapper">
+      <input class="a-input" type="range">
+      <div class="a-input__progress"></div>
+      <div class="a-input__thumb"></div>
+    </div>
+
+h4 Disabled
+.example.-mb--3
+  .-p--3
+    .a-inputWrapper
+      input(class="a-input", type="range", disabled)
+      .a-input__progress
+      .a-input__thumb
+  :code(lang="html")
+    <div class="a-inputWrapper">
+      <input class="a-input" type="range" disabled>
+      <div class="a-input__progress"></div>
+      <div class="a-input__thumb"></div>
+    </div>
+
+h4 With Labels
+.example.-mb--3
+  .-p--3
+    .m-formGroup
+      label.-text.-mr--2 200GB
+      .a-inputWrapper.-w--100
+        input(type="range", class="a-input")
+        .a-input__progress
+        .a-input__thumb
+      label.-text.-ml--2 5TB
+  :code(lang="html")
+    <div class="m-formGroup">
+      <label class="-text -mr--2">200GB</label>
+      <div class="a-inputWrapper -w--100">
+        <input class="a-input" type="range">
+        <div class="a-input__progress"></div>
+        <div class="a-input__thumb"></div>
+      </div>
+      <label class="-text -ml--2">5TB</label>
+    </div>
+
+
+h3 Radio buttons as range slider
```

Comment:
Can we present the JS examples with Javascript? I understand why they're not but to the first time viewer, these examples look broken